lib/request.js

```diff
@@ -1,6 +1,6 @@
 module.exports = request
 
-const isPlainObject = require('is-plain-object')
+const isPlainObject = require('is-plain-object/index.cjs.js')
```

Comment:
This should resolve the cjs version of the library that you were depending on before `is-plain-object@3`. Importing that seems like a better idea than the weird hacky IIFE, and hopefully it'll be possible to resolve this in a way that doesn't require any of this.

Reply:
Yes, great, thank you!